src/constants/links.ts

```diff
@@ -0,0 +1,21 @@
+// Internal Links
+export const INTERNAL_LINKS = {
+  createPolicies: "/kuadrant/all-namespaces/policies",
+  apiDesignerOperator: (namespace: string) => 
```

Comment:
This is TBD IMHO - I don't think we'll have an operator for some time, likely a docs link